src/app/Http/Controllers/AppController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Inertia\Inertia;
+
+class AppController extends Controller
+{
+    public function index() {
+        return Inertia::render('terminal-dashboard');
```

Comment:
Miss de page verplaatsen naar -> /pages/teminal/index.vue. Kunnen we zometeen alle pagina's van de terminal in zn eigen mapje plaatsten.